src/main/java/pl/edu/mimuw/chess/Position.java

```diff
@@ -0,0 +1,30 @@
+package pl.edu.mimuw.chess;
+
+public class Position {
+    private final int row;
+    private final int column;
+    public static boolean validPosition(Position position) {
+        if((position.row < 8) && (position.row >= 0) && (position.column < 8) && (position.column >= 0)) return true;
```

Comment:
ahh, tak się nie pisze, bo to sie nie skaluje, tak próbuj
```
if(position.row >= 8) return false;
if(position.row < 0) return false;
if(position.column >= 8) return false;
if(position.column < 0) return false;
return true;
```
tylko jeszcze stałe wudzielić na te wartości